modules/hosted-control-planes-version-support.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-version-support_{context}"]
+= Versioning support for hosted control planes
+
+//:FeatureName: Hosted control planes
+//include::snippets/technology-preview.adoc[]
+
+A few components of hosted control planes for {product-title} might require different versioning and support levels. The components include the HyperShift Operator, the Control Plane Operator, the command-line interface (CLI), and the API.
```

Comment:
This makes me think that each of them can have a different release cadence. However we can simplify by saying there multiple components that can fall under just two possible release cadences:
1 - Hypershift Operator, HostedCluster/NodePool API, CLI. (one single release cadence)
2 - CPO. (one single release cadence)

Reply:
Thanks, @enxebre. That makes sense. The [upstream docs](https://hypershift-docs.netlify.app/reference/versioning-support/) that I based this topic on uses the terms "a payload release of OCP" and "a release of OCP" to describe the two different cadences. However, the [OCP docs](https://docs.openshift.com/dedicated/osd_architecture/osd_policy/osd-life-cycle.html) use these terms to describe releases:

- Major version
- Minor version
- Patch version

Can you let me know what terms I should use here to describe the two release cadences? For example, would what we have been calling a _payload release_ be the same thing as a major release?

Reply:
Thanks for the clarification, @zanetworker! I'm still a little confused about the concept of the CPO being built and released _in_ an OCP release. I assume that means that the CPO is released on its own schedule apart from the OCP x.y.z release schedule. If that's correct, is there a specific time during the life cycle of a release that the CPO is built/released?

Reply:
I don't understand this well enough to properly explain it in the docs yet, but after I get some clarity, I think we should provide an example to show how the two different release cadences work. 

Something like, "For example, when OCP version 4.12 is released, the HyperShift Operator, CLI, and API resources are released. However, the Control Plane Operator is released _when XYZ happens_." (I'd update that last part after I understand this better.)

Reply:
So the CPO is released as part of the OCP payload, but this term is not customer facing. What customers care about is OCP major/minor/patch. I.e., CPO is part of OpenShift while HyperShift operator runs on OpenShift. Let me know if you are still confused and let's chat :)

Reply:
We can probably skip the part about how the CPO is released because its internal implementation details.